core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BucketDeletionMonitor.scala

```diff
@@ -15,15 +15,15 @@ import scala.concurrent.duration._
 import cats.implicits._
 
 object BucketDeletionMonitor {
-  def props(datasource: SlickDataSource, gcsDAO: GoogleServicesDAO, initialDelay: FiniteDuration, pollInterval: FiniteDuration)(implicit executionContext: ExecutionContext): Props = {
+  def props(datasource: SlickDataSource, gcsDAO: GoogleServicesDAO, initialDelay: FiniteDuration, pollInterval: FiniteDuration)(implicit executionContext: ExecutionContext, cs: ContextShift[IO]): Props = {
     Props(new BucketDeletionMonitor(datasource, gcsDAO, initialDelay, pollInterval))
   }
 
   sealed trait BucketDeletionsMessage
   case object CheckAll extends BucketDeletionsMessage
 }
 
-class BucketDeletionMonitor(datasource: SlickDataSource, gcsDAO: GoogleServicesDAO, initialDelay: FiniteDuration, pollInterval: FiniteDuration)(implicit executionContext: ExecutionContext) extends Actor with LazyLogging {
+class BucketDeletionMonitor(datasource: SlickDataSource, gcsDAO: GoogleServicesDAO, initialDelay: FiniteDuration, pollInterval: FiniteDuration)(implicit executionContext: ExecutionContext, cs: ContextShift[IO]) extends Actor with LazyLogging {
```

Comment:
why did BucketDeletionMonitor change?

Reply:
later version of `cats-effect` requires `ContextShift[F]` for `IO.fromFuture(xxx)`